tests/python/contrib/test_ethosu/test_codegen.py

```diff
@@ -1084,18 +1084,27 @@ def squeeze_func(x):
 
 
 @pytest.mark.parametrize("accel_type", ACCEL_TYPES)
+@pytest.mark.parametrize("half_pixel", [False, True])
```

Comment:
This adds quite a few tests due to an extra dimension in the combinations, it's probably better to add `half_pixel` to certain tests in the `ifm_shape,size` parameterization

Reply:
Agree, I've updated tests.